src/abstractdataframe/reshape.jl

```diff
@@ -226,33 +266,59 @@ unstack(df::AbstractDataFrame, rowkey::ColumnIndex,
 unstack(df::AbstractDataFrame, colkey::ColumnIndex, value::ColumnIndex) =
     unstack(df, index(df)[colkey], index(df)[value])
 
-function unstack(df::AbstractDataFrame, colkey::Int, value::Int)
-    # group on anything not a key or value:
-    rowkeys = setdiff(_names(df), _names(df)[[colkey, value]])
+# group on anything not a key or value
+unstack(df::AbstractDataFrame, colkey::Int, value::Int) =
+    unstack(df, setdiff(_names(df), _names(df)[[colkey, value]]), colkey, value)
+
+unstack(df::AbstractDataFrame, rowkeys, colkey::ColumnIndex, value::ColumnIndex) =
+    unstack(df, rowkeys, index(df)[colkey], index(df)[value])
+
+unstack(df::AbstractDataFrame, rowkeys::AbstractVector{T}, colkey::Int, value::Int) where T<:Real =
```

Comment:
Use `AbstractVector{<:Real}`.

Reply:
OK. Out of curiosity - why it is a preferred style?

Reply:
I think because it's shorter, you can read it directly instead of looking for `T` at the end of the line, and it makes it clear `T` isn't used in the body.

Reply:
👍